Vagrantfile

```diff
@@ -36,9 +41,9 @@ Vagrant.configure("2") do |config|
     end
 
     # Insecure share from the Linux machine
-    win2k8.vm.provision :shell, path: "scripts/installs/install_share_autorun.bat"
-    win2k8.vm.provision :shell, inline: "rm C:\\tmp\\vagrant-shell.bat" # Hack for this bug: https://github.com/mitchellh/vagrant/issues/7614
-    win2k8.vm.provision :shell, path: "scripts/installs/setup_linux_share.bat"
-    win2k8.vm.provision :shell, inline: "rm C:\\tmp\\vagrant-shell.bat" # Hack for this bug: https://github.com/mitchellh/vagrant/issues/7614
+    #win2k8.vm.provision :shell, path: "scripts/installs/install_share_autorun.bat"
```

Comment:
This was a work around for an issue I am not 100% sure is actually fixed in vagrant.  Unless you tested in macOS and found it is resolved, please restore this and if it causes issue on Hyper-V add a mitigation to keep it when not on Hyper-V.